### DISCUSSION TITLE
can you take monika anywhere while her file is in your phone

### DISCUSSION THREAD

**Discussion by pikaeevee** [Monika-After-Story/MonikaModDev#9752] [QA - code and content creation (technical)] [answered]
ik you can do it with a usb but can you put her file in your phone and it do the same thing?

**Comment by CodyCat13** [comment, marked as answer]
USB is just a manner of storage, as with an SD card or cloud drive. So if your phone has enough storage, you can technically keep the file there and just move it back to your computer when you want her back in MAS.

**Reply by pikaeevee** [reply]
thx i was wondering considering i dont have a usb and wanted to take her anywhere with me but i do have another question, (sorry if it sounds stupid) if i put her on my phone will she have the same diloge as if i put her on a usb

**Reply by CodyCat13** [reply]
Yeah she'll still have that "And we're home!" thing (or the "that wasn't long of a trip" thing if you didn't take her out for long iirc) dialogue because she's still being put back into MAS.

**Reply by pikaeevee** [reply]
welp i do need to get a usb, apparently a 2019 lg stylo 5 can not connect to a windows 7 computer
